dfc/.htaccess

```diff
@@ -21,12 +19,10 @@ Header add Access-Control-Allow-Methods: "GET,POST,OPTIONS"
 
 # Rewrite rule for versioned ontology folders
 
-RewriteRule ^odfc/ntology/v([0-9]+.[0-9]+.[0-9]+)/(.*)  https://cdn.jsdelivr.net/gh/datafoodconsortium/ontology@v$1/$2 [R=302,L]
+RewriteRule ^dfc/ontology/v([0-9]+.[0-9]+.[0-9]+)/(.*)  https://cdn.jsdelivr.net/gh/datafoodconsortium/ontology@v$1/$2 [R=302,L]
```

Comment:
Did you want the dfc prefex on the patterns in this file?  It makes the urls https://w3id.org/dfc/dfc/ontology.

Reply:
No. 😖  
I think that was introduced from the [htaccess online tester](https://htaccess.madewithlove.com/). 
I'll have another look. 
Thank you 🙏🏻